doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -257,20 +264,21 @@ Some operations imply that only a partial representation of an entity will be pr
 
 * `id` and `type` are not allowed in update operations, as they are inmutable properties.
 
-* In the requests in which entity `type` is allowed, it may be omitted. When omitted, the default string value `none`
-  is used for the type.
```

Comment:
The user is allowed to use `none` as type at entity creation time, eg:

```
POST /v2/entities?options=keyValues
{
  "id": "E",
  "type": "none",
  "foo": "bar"
}
```

After this POST, a entity with [id, type] = [E, none] is created.

The user can omit time in the above operation, i.e.:

```
POST /v2/entities?options=keyValues
{
  "id": "E",
  "foo": "bar"
}
```

in which case the default applies, and the result of the above POST is the creation of an entity a entity with [id, type] = [E, none] (as in the first example).

Thus, I would say that the answer to your questions is "yes" and "either we treat the `none` as if the type has no type (like if it was omitted)".

Probably this could be more explicit in the specification. Any alternative (i.e. a short paragraph describing this behaviour)?